src/libraries/System.IO.Compression.ZipFile/src/System/IO/Compression/ZipFile.Create.cs

```diff
@@ -351,11 +351,11 @@ public static ZipArchive Open(string archiveFileName, ZipArchiveMode mode, Encod
         ///     otherwise an <see cref="ArgumentException"/> is thrown.</para>
         /// </param>
         public static void CreateFromDirectory(string sourceDirectoryName, string destinationArchiveFileName,
-                                               CompressionLevel compressionLevel, bool includeBaseDirectory, Encoding entryNameEncoding) =>
+                                               CompressionLevel compressionLevel, bool includeBaseDirectory, Encoding? entryNameEncoding) =>
             DoCreateFromDirectory(sourceDirectoryName, destinationArchiveFileName, compressionLevel, includeBaseDirectory, entryNameEncoding: null);
```

Comment:
@stephentoub Looks like there's a bug here. We're not passing `entryNameEncoding` to the called function. Should I just fix it here?

Reply:
Can you do it in a separate PR that includes tests?

Reply:
ok